layouts/shortcodes/cncf-landscape.html

```diff
@@ -57,7 +57,7 @@
 </script>
 {{- end -}}
 <div id="frameHolder">
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.3.9/iframeResizer.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.3.9/iframeResizer.min.js" intregrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" crossorigin="anonymous"></script>
```

Comment:
The `integrity` hash value added in the script tag seems to be incorrect _(along with a typo)_, as I noticed that generating integrity hash locally yields a different value. It appears that the value was simply copied from an existing **_jquery-3.6.0.min.js_**. I suggest trying the following:

```suggestion
  <script src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.3.9/iframeResizer.min.js" integrity="sha384-hHTwgxzjpO1G1NI0wMHWQYUxnGtpWyDjVSZrFnDrlWa5OL+DFY57qnDWw/5WSJOl" crossorigin="anonymous"></script>
```